app/views/console.html

```diff
@@ -89,7 +89,7 @@ <h2  class="query-result-container__header">{{ 'CONSOLE.RESULT_FROM_QUERY' | tra
                     <pre class="query-result {{ resultHeaderTypes[$index] | columnTypeClass }}" ng-if="ColumnTypeCheck.isTimestamp(resultHeaderTypes[$index])">{{ column }} <span class="formatted-timestamp">({{ column | formatTimestamp }})</span></pre>
                     <pre class="query-result {{ resultHeaderTypes[$index] | columnTypeClass }}" ng-if="ColumnTypeCheck.requiresArrayFormatting(resultHeaderTypes[$index])"><formatted-array array="column" typesarray="resultHeaderTypes[$index]" expand="true"></formatted-array></pre>
                     <pre class="query-result {{ resultHeaderTypes[$index] | columnTypeClass }}" ng-if="ColumnTypeCheck.requiresJSONFormatting(resultHeaderTypes[$index])"><formatted-object object="column" expand="true"></formatted-object></pre>
-                    <pre class="query-result {{ resultHeaderTypes[$index] | columnTypeClass }}" ng-if="ColumnTypeCheck.isGeoarea(resultHeaderTypes[$index])"><span>Type</span>: {{ column['type']}}<br/><span>Coordinates</span>: {{ column['coordinates'] }}</pre>
+                    <pre class="query-result {{ resultHeaderTypes[$index] | columnTypeClass }}" ng-if="ColumnTypeCheck.isGeoarea(resultHeaderTypes[$index])"><span>Type</span>: {{ column['type']}} <a href="http://geojson.io/#data=data:application/json,{{ column | formatGeoshape }}" target="_blank"><i class="fa fa-external-link"></i></a><br/><span>Coordinates</span>: {{ column['coordinates'] }} </pre>
```

Comment:
you should use ng-href instead of href:
https://docs.angularjs.org/api/ng/directive/ngHref